ci/cloudbuild/builds/lib/integration.sh

```diff
@@ -28,7 +28,8 @@ source module ci/lib/io.sh
 
 # To run the integration tests we need to install the dependencies for the storage emulator
 export PATH="${HOME}/.local/bin:${PATH}"
-python3 -m pip install --quiet "git+https://github.com/googleapis/storage-testbench@v0.9.0"
+rm -fr "${HOME}/.local"
```

Comment:
This looks dangerous under `ci/cloudbuild/build.sh --local`.

Reply:
Good catch, PTAL.